lib/ASRouterPreferences.jsm

```diff
@@ -50,6 +56,17 @@ class _ASRouterPreferences {
     }, []);
   }
 
+  _migratePrefs() {
```

Comment:
Can you add a comment about what version this is happening and when it should be removed if ever (maybe we should file a ticket for that also)

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1531734

Reply:
Actually, one more thing I am wondering here – if a user preference is already set for the new pref, do we still want to override it with the old one? Probably not right?

Reply:
Wouldn't the migration code run before the user can do any changes to the prefs. I don't understand how we could get in that scenario.

Reply:
Maybe syncing an old profile for example – it's definitely an edge case though